### DISCUSSION TITLE
Is there a way to implement the Block interface?

### DISCUSSION THREAD

**Discussion by Flamgop** [Minestom/Minestom#641] [Help] [answered]
I wanna add some mod support and also wanna see what happens lol

**Comment by TheMode** [comment, marked as answer]
No. The only way to register new blocks is by adding them in the registry. Notice `blocks.json` in your jar archive.

We will in the future introduce an API to modify the registry, but it is not there yet.

**Reply by Flamgop** [reply]
oh makes sense I didn't notice that you could add them to the registry with a json file.
